docs/content/manual/pre-release/node/node-deletion.md

```diff
@@ -11,22 +11,26 @@ title: Test node deletion
 ### Case 2: Delete nodes when there are running volumes:
 1. Deploy Longhorn.
 2. For each node that will be deleted later: 
-    1. create and attach 4 volumes:
+    1. Create and attach 5 volumes:
         1. The 1st volume contains 1 replica only. Both the engines and the replicas are on the pre-delete node. (Attached to the pre-delete node.)
         2. The 2nd volume contains 1 replica only. The engine is on the pre-delete node and the replica is on another node. (Attached to the pre-delete node.)
-        3. The 3rd and the 4th volume contain 3 replicas. Both volumes are attached to a node except for the pre-delete node.
+        3. The 3rd volume contains 1 replica only. The replica is on the pre-delete node and the engine is on another node. (Attached to a node except for the pre-delete node.)
+        4. The 4th volume contain 3 replicas. Attached to a node except for the pre-delete node.
+        5. The 5th volume contain 3 replicas. Attached to a node except for the pre-delete node.
```

Comment:
Why need to have 4 & 5, they look identical?

Reply:
These 2 volumes has identical init state but later on they will be used for different tests

Reply:
Hi @shuo-wu , updated! thank you.